actions/siege

```diff
@@ -2,17 +2,8 @@
 
 set -eux
 
+apt-get install -y python-pip && pip install charm-benchmark
```

Comment:
do a `pip install -U` so that you also get upgrades if you're doing multiple benchmark runs